### PR TITLE
Move tour to overview

### LINKED ISSUE #3276: Move the "Hello World" section from Tour of C# to the new Overview section
This new page will take the updated "Hello World" section from the Tour of C#

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #3276 (finally)

- Update the TOC to move the "tour of C#" under the overview node.
- Remove the existing "get started" index. It was an old landing page.
- Update the remaining "Introduction to C# and .NET Framework" article for .NET 5 and architectural changes.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/csharp/getting-started/?branch=pr-en-us-21066)

### CODE REVIEW

File: docs/csharp/getting-started/index.md [hunk 1]
```diff
@@ -1,51 +1,47 @@
 ---
-title: Get Started - C# Guide
-description: Find short, simple tutorials that let you quickly learn introductory C# concepts and write .NET Core applications.
-helpviewer_keywords:
-  - "Visual C#, getting started"
-  - "getting started, Visual C#"
-ms.date: 04/05/2019
+title: Get Started - an introduction to the C# language and .NET"
+description: Learn the basics of C# and .NET. Get an overview of the C# language and .NET ecosystem.
+ms.date: 10/13/2020
```
Comment (Youssef1313) [inline docs/csharp/getting-started/index.md:4]: Are the helpviewer_keywords intentionally removed?
I assume they have something to do with SEO

Comment (BillWagner) [reply docs/csharp/getting-started/index.md:4]: They were intentionally removed. They were historically used for search in the offline CHM versions of Visual Studio help. They aren't needed any more.

File: docs/csharp/getting-started/index.md [hunk 2]
```diff
@@ -1,51 +1,47 @@
...
-- [Get started with C# and Visual Studio Code](../../core/tutorials/with-visual-studio-code.md)
+- For a good general introduction to the language, see the [Tour of C#](../tour-of-csharp/index.md).
+- For detailed information about specific aspects of the C# language, see the [C# Reference](../language-reference/index.md).
+- For more information about LINQ, see [LINQ (Language-Integrated Query)](../programming-guide/concepts/linq/index.md).
 
-   Visual Studio Code is a free code editor optimized for building and debugging modern web and cloud applications. It supports IntelliSense and is available for Linux, macOS, and Windows.
+## .NET Platform Architecture
 
-   This topic shows you how to create and run a simple Hello World application with Visual Studio Code and .NET Core.
+C# programs run on .NET, a virtual execution system called the common language runtime (CLR) and a unified set of class libraries. The CLR is the commercial implementation by Microsoft of the common language infrastructure (CLI), an international standard. The CLIis the basis for creating execution and development environments in which languages and libraries work together seamlessly.
```
Comment (IEvangelist) [inline docs/csharp/getting-started/index.md:37]:
```suggestion
C# programs run on .NET, a virtual execution system called the common language runtime (CLR) and a unified set of class libraries. The CLR is the commercial implementation by Microsoft of the common language infrastructure (CLI), an international standard. The CLI is the basis for creating execution and development environments in which languages and libraries work together seamlessly.
```

File: docs/csharp/getting-started/index.md [hunk 3]
```diff
@@ -1,51 +1,47 @@
...
 
-    Provides information about C# programming concepts, and describes how to perform various tasks in C#.
+Language interoperability is a key feature of .NET. IL code produced by the C# compiler conforms to the Common Type Specification (CTS). IL code generated from C# can interact with code that was generated from the .NET versions of F#, Visual Basic, C++, or any of more than 20 other CTS-compliant languages. A single assembly may contain multiple modules written in different .NET languages, and the types can reference each other as if they were written in the same language.
 
-- [C# Reference](../language-reference/index.md)
+In addition to the run time services, .NET also includes an extensive library of over 4000 classes. The .NET library is organized into namespaces that provide a wide variety of useful functionality for everything from file input and output to string manipulation to XML parsing, to web application frameworks to Windows Forms controls. The typical C# application uses the .NET class library extensively to handle common "plumbing" chores.
```
Comment (IEvangelist) [inline docs/csharp/getting-started/index.md:45]:
```suggestion
In addition to the run time services, .NET also includes an extensive library of over 4,000 classes. The .NET library is organized into namespaces that provide a wide variety of useful functionality for everything from file input and output to string manipulation to XML parsing, to web application frameworks to Windows Forms controls. The typical C# application uses the .NET class library extensively to handle common "plumbing" chores.
```

Is 4,000 classes accurate - I'd imagine that it's WAY more now? Also, you mention the ".NET library" here as if there is only one, which one are you referring to - I'm confused by that? Do you mean the runtime? If so the, there are multiple implementations...I'm not sure how to update this. Maybe a link to the library?

**Review by IEvangelist — APPROVED** [review]
I had a few questions and suggestions. Looks good to me though, :shipit: when you're ready.

**Pull request merged**